tests/test_models/test_forward.py

```diff
@@ -519,6 +519,7 @@ def test_yolact_forward():
         gt_masks=gt_masks,
         return_loss=True)
     assert isinstance(losses, dict)
+    detector.forward_dummy(imgs)
```

Comment:
Add a comment to clarify what this line is testing.
Also, add some assert to test the output.

Reply:
I am following example of testing `forward_dummy` in `test_forward.py` `test_sparse_rcnn_forward` L83
```
assert isinstance(losses, dict)
loss, _ = detector._parse_losses(losses)
assert float(loss.item()) > 0
detector.forward_dummy(imgs)
```
What's the requirements of outputs of `forward_dummy`?